deployments/fs.go

```diff
@@ -0,0 +1,19 @@
+package deployments
+
+import (
+	"embed"
+	"io/fs"
+)
+
+// By default, go embed does not embed files that starts with `.` or `_` that's why we need to add _helpers.tpl explicitly
+
+//go:embed charts/* charts/kuma/templates/_helpers.tpl
```

Comment:
This will also include things like the `.gitignore`, `.helmdocsignore`, and `.helmignore` as well as the other "useless" files like the readme(s). Do you think we should use the * glob for the templates, and everything else explicitly? 


```suggestion
//go:embed charts/kuma/templates/* charts/kuma/Chart.yaml charts/kuma/values.yaml
```

Reply:
Thanks, you are right, we can save a bit KBs by not embedding useless files. I changed the pattern